ml_pipeline/scripts/training.py

```diff
@@ -0,0 +1,76 @@
+import os
+import json
+import pickle
+import pandas as pd
+import numpy as np
+import rasterio as rio
+import geopandas as gpd
+from pycaret.classification import *
+import argparse
+
+def load_config(config_path):
+    with open(config_path, 'r') as config_file:
+        config = json.load(config_file)
+    return config
+
+def train_with_config(config):
+    gdf = gpd.read_file(config['Paths']['shapefile_path'])
+    new_df1 = gdf.drop(columns=config['ColumnsToDrop'])
+    
+    new_df1 = new_df1.rename(columns=lambda col: extract_number(col))
```

Comment:
Is `extract_number` defined in `pycaret.classification`? Or elsewhere? I can't see it defined in your code, or in pycaret :thinking: :
https://github.com/pycaret/pycaret/blob/master/pycaret/classification/__init__.py
https://pycaret.readthedocs.io/en/stable/api/classification.html